### PR TITLE
Add toolbar and headline sections

### PR DESCRIPTION
**Pull request by biruk-bereka**
Add toolbar section:
       - Add Name and Menu Icon for the navbar
  Add headline section:
       Add three different content to the welcome page:
       - The name of the person
       - The profession of the person
       - The professional experience

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,58 @@
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
-    <h1>My Portfolio Project</h1>
+    <header>
+        <nav class="nav-bar">
```
Comment (Hamzaoutdoors) [inline index.html:13]: - [ ] You're including both the logo and the hamburger icon in the `<nav>` tag, but it should only include elements that include navigation elements. As only the hamburger element will contain the menu items later on in the project, please change your codebase to include only the hamburger icon in the `<nav>` tag. The logo is fine as it is, good job making it a link to the homepage 👍

File: index.html [hunk 2]
```diff
@@ -8,6 +8,58 @@
...
+            <h1 class="name">Biruk Bereka</h1>
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ]  It's required that we enclose this Logo in an anchor `<a>` tag. Please consider refactoring it. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of UX convention make the navigation in the website way more pleasant and intuitive.

File: index.html [hunk 3]
```diff
@@ -8,6 +8,58 @@
...
+            <div><img src="./Icons/Icon-Menu.png" alt=""></div>
+        </nav>
+    </header>
+    <main>
+        <section class="welcome-section">
+            <article class="content-1-article">
+              <h2 class="content-1">Hey There. I’m Biruk</h2>
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ] The main title (e.g., "I'm Andrew...") is a `h1` tag, Kindly, refactor that .. that will improve your project to look more semantic 😄

File: index.html [hunk 4]
```diff
@@ -8,6 +8,58 @@
...
+            </article>
+            <article class="content-2-article">
+              <h3 class="content-2">I am a Software Developer</h3>
+            </article>
```
Comment (Hamzaoutdoors) [inline index.html]: - [OPTIONAL] To make your code less longer and more readable you can get rid of `article` tag from here ^^

File: index.css [hunk 5]
```diff
@@ -1,4 +1,95 @@
-body {
-  padding: 0;
```
Comment (Hamzaoutdoors) [inline index.css]: - [ ] Please make sure you're following the template's margin and padding. At the moment, you have an extra margin, kindly fix this issue by adding `margin: 0;` to the `body` to better match the template.

real

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @biruk-bereka, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @hamzaoutdoors in your question so I can receive the notification ^_^

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ReshyResh — APPROVED** [review]
# Hi @biruk-bereka , 

### Good job addressing the changes requested by the previous reviewer!
#### Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**